### PR TITLE
Docs/bai 376 administration docs

### PR DESCRIPTION
**Pull request by ARADDCC011**

### CODE REVIEW

File: pages/docs/administration/getting-started/configuration/making-a-schema.mdx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+priority: 200
+---
+
+import DocsWrapper from '@/src/docs/DocsWrapper'
+
+# Making a Schema
+
+export default ({ children }) => <DocsWrapper>{children}</DocsWrapper>
+
+- The schemas use a JSON schema (existing examples are in server/scripts/example_schemas)
```
Comment (ARADDCC002) [inline pages/docs/administration/getting-started/configuration/making-a-schema.mdx:11]: This needs to be expanded on much more. 

We should be detail why each property is used (like we are with widget) and also give other examples. Examples of how to make multiple choice, nested pages, dependencies, etc. 

We don't need to go into HUGE amounts of detail as we can forward users to the library's documentation, but as a user I'd expect a quick guide on how to make a very small schema from scratch.

File: pages/docs/administration/getting-started/helm-deployments.mdx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+export default ({ children }) => <DocsWrapper>{children}</DocsWrapper>
+
+## Note
+
+Below deployment is configured for OpenShift. You will have to change route configuraiton to ingress for k8s.
+
+## Prerequisite
+
+- OpenShift CLI: https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/linux/oc.tar.gz
```
Comment (a3957273) [inline pages/docs/administration/getting-started/helm-deployments.mdx:17]: Use the Markdown format for links, `[name](http://example.com)`

File: pages/docs/administration/getting-started/building-docker-image.mdx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+Below lists options to change application configuration during dockerbuild.
+
+- default application configuration is applied from config/default.js.
+- production.js overwrites mongo configuration.
+- production.js overwrites minio configuration.
+- production.js overwrites registry configurration.
+- build.environment: openshift //must be set for OpenShift
+- openshift.[ namespace | appPublicRoute ] //must be set for OpenShift
+- smtp.enabled: true //must be set, else Bailo does not start the s2i build
+- logging.file.enabled: false //set true to debug Bailo
```
Comment (a3957273) [inline pages/docs/administration/getting-started/building-docker-image.mdx:26]: None of these actually affect the built Docker image.  These should all be removed.

Comment (a3957273) [reply pages/docs/administration/getting-started/building-docker-image.mdx:26]: These are all configured at runtime!

File: pages/docs/administration/getting-started/configuration/app-configuration.mdx [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+
+# App Configuration
+
+export default ({ children }) => <DocsWrapper>{children}</DocsWrapper>
+
+## Note
+
+Below listed configurations can be changed during dockerbuild.
+
+- default application configuration is applied from config/default.js.
```
Comment (a3957273) [inline pages/docs/administration/getting-started/configuration/app-configuration.mdx:15]: These are many more, it's odd to pick out just these few options.

Comment (a3957273) [reply pages/docs/administration/getting-started/configuration/app-configuration.mdx:15]: I'd argue these are either obvious or should be documented within the config themselves.  Not here.

File: pages/docs/administration/getting-started/configuration/app-configuration.mdx [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+- smtp.enabled: true //must be set, else Bailo does not start the s2i build
+- logging.file.enabled: false //set true to debug Bailo
+
+## production.js
+
+production.js can be overwritten during HELM deployment. See helm/bailo/templates/bailo/bailo.configmap.yaml
+
+## Example **default.js**
+
+```
```
Comment (a3957273) [inline pages/docs/administration/getting-started/configuration/app-configuration.mdx:30]: This isn't an example, this _is_ the default configuration.  It shouldn't be included here, as it changes often.  Instead, you should link / direct the user to the location.

You should instead talk about `node-config` and how it overrides files (including the order of those overrides).

**Review by a3957273 — CHANGES_REQUESTED** [review]

File: pages/docs/administration/getting-started/building-docker-image.mdx (hunk 3 above)
Comment (ARADDCC013) [reply pages/docs/administration/getting-started/building-docker-image.mdx]: amended - removed as per comment.

File: pages/docs/administration/getting-started/configuration/app-configuration.mdx (hunk 4 above)
Comment (ARADDCC013) [reply pages/docs/administration/getting-started/configuration/app-configuration.mdx]: added link to node config docs page and reference to helm making changes to application during runtime.

File: pages/docs/administration/getting-started/helm-deployments.mdx (hunk 2 above)
Comment (ARADDCC013) [reply pages/docs/administration/getting-started/helm-deployments.mdx]: complete.

**Review by a3957273 — APPROVED** [review]

**Pull request merged**